examples/no-prefix/variables.tf

```diff
@@ -0,0 +1,93 @@
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to provision resources to"
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "us-south"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
```

Comment:
well this is not actually true - its used for all resources except the ones created in the VPC module itself.